_posts/2016-12-06-recurse-center-week-10.md

```diff
@@ -0,0 +1,122 @@
+
+Doing another course doesn't feed my need to accomplish a big project while I'm here - but I can build a project in Haskell after RC.
+
+#### **Interview Preps**
+
+This past week I did 2 interview prep sessions. These have been super helpful.
+
+I've done interview prep with friends in the past, but nothing to the level of detail done at RC. Since first starting interview prep at RC, I've started following a system for whiteboard interviews.
+
+Some things I'm trying to do:
+
+- Restate the problem
+- Write down assumptions
+- Write out an example input and output
+- Brainstorm out loud possible solutions and write them down (noting the tradeoffs)
+- Write out the API for my code (eg. "I'm going to write a function `X` that will accept an object with properties `a`, `b` and `c`").
+- Be as verbose as possible with the doubts that are getting me stuck. Ask questions.
+- Create test cases and walk through the final implementation.
+
+This is way better than my previous strategy:
+
+- Code a solution while being as verbose as possible.
+- Walk through it after once the interviewer asks you to.
+
+#### **Advent of Code**
+
+I've passively been working through [Advent of Code](http://adventofcode.com/). Think of it as a Advent Calendar that hands out difficult coding problems instead of delicious chocolate.
+
+It's fun, and many people are doing it which leads to good discussion. On the other hand, I've been feeling less enthused about random coding problems. I'm more interested in working on projects.
+
+#### **Essentialism**
+
+I've started reading "[Essentialism: The Disciplined Pursuit of Less](https://www.goodreads.com/book/show/18077875-essentialism)".
+
+Ironically, I think reading about time management is non-essential - but I'm going to put that aside and pretend it's the book teaching me that.
+
+In the past week I've been thinking a lot about what is essential and what has little value. Sometimes I think of things to cut away, without any immediate idea of what to replace them with. My plan is to fill a lot more of my time journalling and meditating.
+
+### Physical
+
+#### **Park walk**
+
+On Sunday I went for a walk through Prospect Park with some housemates. It was nice to be away from computers and technology and just stroll about. Fresh air always makes me feel better.
+
+#### **Blob**
+
+Days on end of sitting at a computer is really starting to make me feel out of shape. REALLY looking forward to settling in somewhere and being able to do a better job of focusing on my health.
+
+### </End>
+
+Woo! We did it - and it's not even Friday! Getting back on track 😊.
+
+While I may not be feeling like a massively better programmer at the moment, I sure feel like a much better person. I was thinking about how much time I've invested in blogging. I think it's helped maximize what I get out of all these discussions and questions. So that's been nice!
```

Comment:
Harold's version: "I'm not a better programmer, but I'm a better person!" 

1/2 isn't bad 😛